src/api/types.ts

```diff
@@ -440,6 +440,11 @@ interface TestHelperAccountCreate {
   deleteAccounts?: boolean;
 }
 
+interface TestHelperCreate {
+  context: Context;
+  companyId?: string;
+}
```

Comment:
Can we rename this type to be more descriptive as its not generic, not a blocker.